### DISCUSSION TITLE
collection field/column delete option ?

### DISCUSSION THREAD

**Discussion by TERIIHUB** [pocketbase/pocketbase#3065] [Q&A] [answered]
Hi all,
I cant locate the button to delete a field in the edit form, I can only add or rename one...
Is that a present limitation of pocketbase ?
Thanks for the help

**Comment by ganigeorgiev** [comment, marked as answer]
It is buried under the field options panel:

![field_delete](https://i.imgur.com/7QWRFGt.png)

**Comment by TERIIHUB** [comment]
oh, my...sign of Old-tidude ! lol
Thanks ganigeorgiev
